dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1220,15 +1221,19 @@ protected void launchHelp() {
         out("");
         out("Required parameters:");
         out("  --entry <entry>                     Complete entry path in the Dockstore (ex. quay.io/collaboratory/seqware-bwa-workflow:develop)");
-        out("   OR");
-        out("  --local-entry <local-entry>         Allows you to specify a full path to a local descriptor instead of an entry path");
+        if (!(this instanceof CheckerClient)) {
+            out("   OR");
+            out("  --local-entry <local-entry>         Allows you to specify a full path to a local descriptor instead of an entry path");
+        }
         out("");
         out("Optional parameters:");
         out("  --json <json file>                  Parameters to the entry in the dockstore, one map for one run, an array of maps for multiple runs");
         out("  --yaml <yaml file>                  Parameters to the entry in the dockstore, one map for one run, an array of maps for multiple runs (Only for CWL)");
         out("  --tsv <tsv file>                    One row corresponds to parameters for one run in the dockstore (Only for CWL)");
         out("  --descriptor <descriptor type>      Descriptor type used to launch workflow. Defaults to " + CWL_STRING);
-        out("  --local-entry                       Allows you to specify a full path to a local descriptor for --entry instead of an entry path");
+        if (!(this instanceof CheckerClient)) {
+            out("  --local-entry                       Allows you to specify a full path to a local descriptor for --entry instead of an entry path");
```

Comment:
when this is not a checker client, does it display --local-entry as a possible required parameter (OR) and also an optional parameter?

Reply:
This just maintains the current behavior which is to display it in both places. 
It does seem a bit odd, will create follow-up item to clean up these descriptions